starter-code/index.js

```diff
@@ -1,10 +1,41 @@
-var SortedList = function() {};
+var SortedList = function(items, length) {
+  this.items = [];
+  this.length = 0;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add = function(item) {
+  this.items.push(item);
+  this.items.sort(function(a, b) {
```

Comment:
Now you know how to use nice ES6 syntax like this:
`this.items.sort( (a, b) => a - b )`

:)